packages/vocabularies-types/src/Edm.ts

```diff
@@ -458,7 +458,7 @@ export type ConvertedMetadata = {
     entityTypes: EntityType[];
     references: Reference[];
     diagnostics: { message: string }[];
-    resolvePath: <T>(path: string) => ResolutionTarget<T>;
+    resolvePath: <T>(path: string, dontBeSmart?: boolean) => ResolutionTarget<T>;
```

Comment:
What a wonderful parameter name ...

Reply:
I can probably change that one if you prefer :)